lib/main_containers.pm

```diff
@@ -0,0 +1,71 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: module loader of container tests in SLES
+# Maintainer: qa-c@suse.de
+
+package main_containers;
+use base 'Exporter';
+use Exporter;
+use utils;
+use version_utils;
+use main_common qw(loadtest boot_hdd_image);
+use containers::utils qw(loadtest_containers);
+use testapi qw(check_var get_required_var get_var);
+use Utils::Architectures;
+use strict;
+use warnings;
+
+
+our @EXPORT = qw(
+  load_container_tests
+);
+
+sub load_container_tests {
+    my $runtime = get_required_var('CONTAINER_RUNTIME');
+    boot_hdd_image;
+
+    # Container Image tests
+    if (get_var('FLAVOR', '') =~ /Container-Image/) {
+        loadtest_containers 'host_configuration';
+        loadtest_containers 'docker_image';
+        loadtest_containers 'podman_image';
+    }
+
+    # Container Host tests
+    else {
+        if ($runtime eq 'docker') {
+            loadtest_containers 'docker';
+            unless (is_sle("<=15") and is_aarch64) {
+                loadtest_containers 'zypper_docker';
+                loadtest_containers 'docker_runc';
+            }
+            loadtest_containers 'docker_image';
+            loadtest_containers 'buildah_docker' if is_sle('>=15-SP1');
+            loadtest_containers 'validate_btrfs' if is_x86_64;
```

Comment:
I'd move this module to the end of `docker` tests.

Reply:
ok

Reply:
done